src/resources/js/move-tickets-dialog.js

```diff
@@ -0,0 +1,496 @@
+/**
+ * Handles the move tickets/move ticket type dialogs.
+ *
+ * @var object tribe_move_tickets
+ * @var object tribe_move_tickets_data
+ */
+
+var tribe_move_tickets = ( 'object' === typeof tribe_move_tickets ) ? tribe_move_tickets : {};
+
+( function( $, obj ) {
+	var $main,
+		$processing,
+		$errors,
+		$current_stage,
+		current_stage_name,
+		$stages,
+		$back,
+		$next,
+		last_direction,
+		skip_choose_event,
+		target_post_id,
+		target_ticket_type_id;
+
+
+	function init() {
```

Comment:
The way all of these functions are declared makes it pretty hard to debug/extend/modify/test (I'm just pretending that we have tests for JS as I type this). These functions are essentially all private and untouchable outside the confines of this anonymous block.

Using this approach resolves those issues:

```js
obj.init = function() {
};
```